src/dbus_api/macros.rs

```diff
@@ -84,3 +84,14 @@ macro_rules! properties_footer {
         }
     };
 }
+
+macro_rules! pre_pool_op {
```

Comment:
Given that this macro does things both before and after the pool operation, this is no longer just pre-pool operation. It actually wraps the pool operation. I would recommend something along the lines of `pool_op_logging` or something similar.

Reply:
Resolved; the new name is pool_op_logging!()